File: app/src/main/java/com/prey/events/retrieves/EventRetrieveDataNullMobile.java
/*******************************************************************************
 * Created by Orlando Aliaga
 * Copyright 2019 Prey Inc. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.events.retrieves;

import android.content.Context;
import com.prey.events.manager.EventManager;

import org.json.JSONException;
import org.json.JSONObject;

public class EventRetrieveDataNullMobile {

    public  void execute(Context context, EventManager manager){
        JSONObject mobileSon = new JSONObject();
        try {
            mobileSon.put("mobile_internet", "");
        } catch (Exception e) {
        }
        manager.receivesData(EventManager.MOBILE, mobileSon);
    }
}

File: app/src/main/java/com/prey/events/retrieves/kotlin/EventRetrieveDataNullMobile.kt
/*******************************************************************************
 * Created by Orlando Aliaga
 * Copyright 2025 Prey Inc. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.events.retrieves.kotlin

import android.content.Context
import com.prey.events.manager.kotlin.EventManager
import com.prey.kotlin.PreyLogger
import org.json.JSONObject

class EventRetrieveDataNullMobile {
    fun execute(context: Context?, manager: EventManager) {
        val mobileSon = JSONObject()
        try {
            mobileSon.put("mobile_internet", "")
        } catch (e: Exception) {
            PreyLogger.e("Error:" + e.message, e)
        }
        manager.receivesData(EventManager.MOBILE, mobileSon)
    }
}